test/cbmc/sources/mqtt_agent_cbmc_state.c

```diff
@@ -143,3 +149,36 @@ bool isAgentSendCommandFunctionStatus( MQTTStatus_t mqttStatus )
             ( mqttStatus == MQTTNoMemory ) ||
             ( mqttStatus == MQTTSendFailed ) );
 }
+
+void addPendingAcks( MQTTAgentContext_t * pContext )
+{
+    uint8_t i;
+    uint16_t packetId;
+    Command_t * pCommand;
+    MQTTPublishInfo_t * pPublishInfo;
+
+    __CPROVER_assert( pContext != NULL, "MQTT Agent Context is not NULL" );
+
+    for( i = 0; i < MQTT_AGENT_MAX_OUTSTANDING_ACKS; i++ )
+    {
+        /* Limit the packet Ids so that the range of packet ids retrieved through
+         * MQTT_PublishToResend can be limited as well. */
+        __CPROVER_assume( packetId < MAX_PACKET_ID );
+        pContext->pPendingAcks[ i ].packetId = packetId;
+
+        /*Add a publish command. */
```

Comment:
I think this file needs to be uncrustified

Reply:
```suggestion
        /* Add a publish command. */
```

Reply:
Uncrustify didn't catch this issue. Addressing it as a suggestion.